<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneEditor</class>
 <widget class="QMainWindow" name="SceneEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hair - Scene Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="controlsBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>182</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>182</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>182</width>
      <height>700</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>181</width>
       <height>711</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="testHolder">
      <item>
       <widget class="QPushButton" name="showHideGroupSim">
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>Simulation </string>
        </property>
        <property name="icon">
         <iconset resource="../../images/images.qrc">
          <normaloff>:/images/chevron_up.png</normaloff>
          <normalon>:/images/chevron_down.png</normalon>:/images/chevron_up.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupSim">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>133</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>160</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>166</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Gravity:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputHairRadius_2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>10</y>
           <width>50</width>
           <height>21</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>TODO</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QSlider" name="sliderHairsPerPatch_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>64</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>166</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Wind:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputHairRadius_4">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>50</y>
           <width>50</width>
           <height>21</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>TODO</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QSlider" name="sliderHairsPerPatch_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>151</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>64</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QCheckBox" name="frictionSimCheckBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>187</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Friction Sim</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="showHideGroupTess">
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>Tesselation</string>
        </property>
        <property name="icon">
         <iconset resource="../../images/images.qrc">
          <normaloff>:/images/chevron_up.png</normaloff>
          <normalon>:/images/chevron_down.png</normalon>:/images/chevron_up.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupTess">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>220</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>166</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Spline Verts:</string>
         </property>
        </widget>
        <widget class="QSlider" name="sliderHairRadius">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>150</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputSplineVertices">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>90</y>
           <width>50</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>166</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Hair Radius:</string>
         </property>
        </widget>
        <widget class="QSlider" name="sliderSplineVertices">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>150</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>64</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputHairsPerPatch">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>10</y>
           <width>40</width>
           <height>21</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Hairs per patch:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputHairRadius">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>50</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QSlider" name="sliderHairsPerPatch">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>150</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>64</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>166</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Noise Amp:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputNoiseAmp">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>130</y>
           <width>50</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>166</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Noise Freq:</string>
         </property>
        </widget>
        <widget class="QSlider" name="sliderNoiseAmp">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>150</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="inputHairRadius_5">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>170</y>
           <width>50</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>TODO</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QSlider" name="sliderHairRadius_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>150</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="editScene">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>131</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Edit Scene</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="showHideGroupRender">
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>Rendering  </string>
        </property>
        <property name="icon">
         <iconset resource="../../images/images.qrc">
          <normaloff>:/images/chevron_up.png</normaloff>
          <normalon>:/images/chevron_down.png</normalon>:/images/chevron_up.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupRender">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>181</width>
           <height>100</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Hair Color (rgb)</string>
         </property>
         <widget class="QSlider" name="sliderHairColorR">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
          <property name="maximum">
           <number>2550</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QSlider" name="sliderHairColorG">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
          <property name="maximum">
           <number>2550</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QSlider" name="sliderHairColorB">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
          <property name="maximum">
           <number>2550</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLineEdit" name="inputHairColorR">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>30</y>
            <width>40</width>
            <height>21</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="inputHairColorG">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>50</y>
            <width>40</width>
            <height>21</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="inputHairColorB">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>70</y>
            <width>40</width>
            <height>21</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
        <widget class="QCheckBox" name="shadowCheckBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>187</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Shadows</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="supersampleCheckBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>187</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Supersampling</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>211</width>
       <height>451</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>182</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>600</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>178</width>
        <height>556</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>460</y>
       <width>196</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pauseButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonResetSim">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="buttonResetSim_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>500</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Apply and Close</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
